packages/ckeditor5-ui/tests/button/buttonview.js

```diff
@@ -298,14 +298,6 @@ describe( 'ButtonView', () => {
 				expect( view.element.hasAttribute( 'aria-pressed' ) ).to.be.false;
 			} );
 
-			it( '-checked reacts on #isOn', () => {
```

Comment:
Removing the test doesn't cover the change (against regression). I.e. if `aria-checked` is reintroduced by mistake on button, no test would fail. Please add one that checks if there is no such attribute on button.

Reply:
Added.